collective/googlenews/browser/configure.zcml

```diff
@@ -0,0 +1,27 @@
+<configure xmlns="http://namespaces.zope.org/zope"
+    xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    i18n_domain="collective.googlenews">
+  <configure zcml:condition="not-have plone-43">
+    <browser:page for="collective.syndication.interfaces.ISyndicatable"
```

Comment:
@rodfersou I think we don't need to support Plone 4.2 right now, because it will add a lot of complexity that we don't have time to deal with right now; plese remove any code associated with collective.syndication.